### PR TITLE
Failover database and elastic pool new powershell cmdlets

### PR DESCRIPTION
**Pull request by emgu-ms**
## Checklist

- [X] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [X] The title of the PR is clear and informative
- [X] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [X] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [X] If applicable, the changes made in the PR have proper test coverage
- [X] For public API changes to cmdlets:
    - [X] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [X] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: src/Accounts/Accounts/Accounts.csproj [hunk 1]
```diff
@@ -29,6 +29,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Management.Sql" Version="1.33.0-preview" />
```
Comment (cormacpayne) [inline src/Accounts/Accounts/Accounts.csproj]: @emgu-ms please remove this reference

File: src/Accounts/Authentication.ResourceManager/Authentication.ResourceManager.csproj [hunk 2]
```diff
@@ -11,6 +11,10 @@
     <RootNamespace>$(LegacyAssemblyPrefix)Common.Authentication.ResourceManager</RootNamespace>
   </PropertyGroup>
 
+    <ItemGroup>
+      <PackageReference Include="Microsoft.Azure.Management.Sql" Version="1.33.0-preview" />
```
Comment (cormacpayne) [inline src/Accounts/Authentication.ResourceManager/Authentication.ResourceManager.csproj]: @emgu-ms please revert the changes made in this file

File: src/Accounts/Authentication/Authentication.csproj [hunk 3]
```diff
@@ -12,6 +12,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Management.Sql" Version="1.33.0-preview" />
```
Comment (cormacpayne) [inline src/Accounts/Authentication/Authentication.csproj]: @emgu-ms please remove this reference

File: src/EventHub/EventHub/EventHub.csproj [hunk 4]
```diff
@@ -17,6 +17,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Management.EventHub" Version="2.5.0" />
+    <PackageReference Include="Microsoft.Azure.Management.Sql" Version="1.33.0-preview" />
```
Comment (cormacpayne) [inline src/EventHub/EventHub/EventHub.csproj]: @emgu-ms please remove this reference

File: src/KeyVault/KeyVault/KeyVault.csproj [hunk 5]
```diff
@@ -15,6 +15,7 @@
     <PackageReference Include="Microsoft.Azure.KeyVault" Version="3.0.1" />
     <PackageReference Include="Microsoft.Azure.KeyVault.WebKey" Version="3.0.1" />
     <PackageReference Include="Microsoft.Azure.Management.KeyVault" Version="2.4.2" />
+    <PackageReference Include="Microsoft.Azure.Management.Sql" Version="1.33.0-preview" />
```
Comment (cormacpayne) [inline src/KeyVault/KeyVault/KeyVault.csproj]: @emgu-ms please remove this reference

File: src/Monitor/Monitor/Monitor.csproj [hunk 6]
```diff
@@ -12,6 +12,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Management.Monitor" Version="0.22.0-preview" />
+    <PackageReference Include="Microsoft.Azure.Management.Sql" Version="1.33.0-preview" />
```
Comment (cormacpayne) [inline src/Monitor/Monitor/Monitor.csproj]: @emgu-ms please remove this reference

File: src/Network/Network/Network.csproj [hunk 7]
```diff
@@ -13,6 +13,7 @@
   <ItemGroup>
     <PackageReference Include="AutoMapper" Version="6.2.2" />
     <PackageReference Include="Microsoft.Azure.Management.Network" Version="19.13.0-preview" />
+    <PackageReference Include="Microsoft.Azure.Management.Sql" Version="1.33.0-preview" />
```
Comment (cormacpayne) [inline src/Network/Network/Network.csproj]: @emgu-ms please remove this reference

File: src/OperationalInsights/OperationalInsights/OperationalInsights.csproj [hunk 8]
```diff
@@ -12,6 +12,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Management.OperationalInsights" Version="0.19.0-preview" />
+    <PackageReference Include="Microsoft.Azure.Management.Sql" Version="1.33.0-preview" />
```
Comment (cormacpayne) [inline src/OperationalInsights/OperationalInsights/OperationalInsights.csproj]: @emgu-ms please remove this reference

File: src/Sql/Sql.LegacySdk/Sql.LegacySdk.csproj [hunk 9]
```diff
@@ -13,7 +13,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json" Version="10.0.3" PrivateAssets="all"/>
+    <PackageReference Include="Microsoft.Azure.Management.Sql" Version="1.33.0-preview" />
```
Comment (cormacpayne) [inline src/Sql/Sql.LegacySdk/Sql.LegacySdk.csproj]: @emgu-ms please remove this reference

File: src/Sql/Sql/Az.Sql.psd1 [hunk 10]
```diff
@@ -265,7 +265,9 @@ AliasesToExport = 'Get-AzSqlDatabaseServerAuditingPolicy',
                'Get-AzSqlDatabaseThreatDetectionPolicy', 
                'Set-AzSqlDatabaseThreatDetectionPolicy', 
                'Remove-AzSqlDatabaseThreatDetectionPolicy', 
-               'Set-AzSqlInstanceTDEProtector', 'Get-AzSqlInstanceTDEProtector'
+               'Set-AzSqlInstanceTDEProtector', 'Get-AzSqlInstanceTDEProtector',
+			   'Invoke-AzureSqlDatabaseFailover',
```
Comment (cormacpayne) [inline src/Sql/Sql/Az.Sql.psd1]: @emgu-ms would you mind fixing the identation here? It looks like it's an issue with tabs vs. spaces 😁

File: src/Sql/Sql/ChangeLog.md [hunk 11]
```diff
@@ -24,6 +24,7 @@
 * Fix missing examples for Set-AzSqlDatabaseSecondary cmdlet
 * Fix set Vulnerability Assessment recurring scans without providing any email addresses
 * Fix a small typo in a warining message.
+* Add failover database and elastic pool new cmdlets.
```
Comment (cormacpayne) [inline src/Sql/Sql/ChangeLog.md]: @emgu-ms this line you're adding should go under `## Upcoming Release` -- I believe this happened due to merging issues after the latest release

File: tools/ScenarioTest.ResourceManager/ScenarioTest.ResourceManager.csproj [hunk 12]
```diff
@@ -12,6 +12,7 @@
   </PropertyGroup>
 
   <ItemGroup>    
+    <PackageReference Include="Microsoft.Azure.Management.Sql" Version="1.33.0-preview" />
```
Comment (cormacpayne) [inline tools/ScenarioTest.ResourceManager/ScenarioTest.ResourceManager.csproj]: @emgu-ms please remove this reference

File: tools/TestFx/TestFx.csproj [hunk 13]
```diff
@@ -11,6 +11,10 @@
     <RootNamespace>$(LegacyAssemblyPrefix)TestFx</RootNamespace>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Management.Sql" Version="1.33.0-preview" />
```
Comment (cormacpayne) [inline tools/TestFx/TestFx.csproj]: @emgu-ms please revert the changes made in this file

**Comment by johnpaulkee** [comment]
fyi @emgu-ms some of your managedInstance tests are probably failing due to the Management.Sql version bump. I have fix for that here and am hoping to get it checked in when possible - https://github.com/Azure/azure-powershell/pull/9744

after it's checked in, you should be able to sync with master again and have those managed instance tests not block your build.

**Comment by emgu-ms** [comment]
> fyi @emgu-ms some of your managedInstance tests are probably failing due to the Management.Sql version bump. I have fix for that here and am hoping to get it checked in when possible - #9744
> 
> after it's checked in, you should be able to sync with master again and have those managed instance tests not block your build.

Cool, thanks for the heads up!

**Comment by johnpaulkee** [comment]
> > fyi @emgu-ms some of your managedInstance tests are probably failing due to the Management.Sql version bump. I have fix for that here and am hoping to get it checked in when possible - #9744
> > after it's checked in, you should be able to sync with master again and have those managed instance tests not block your build.
> 
> Cool, thanks for the heads up!

Hey @emgu-ms , my changes have been merged. Feel free to pull from master to grab the latest changes and you should be able to get a successful full test run without MI tests failing.
